app/assets/javascripts/embargo_reason_field.js

```diff
@@ -0,0 +1,11 @@
+$( document ).ready(function() {
+  $('#embargo_reason_select').change(function() {
```

Comment:
So if they dont trigger a change, it accepts the form per usual. I didnt want to make the embargo reason required. What I can do though, if it sounds like a good idea, is to pre-fill the value for them and make not filling embargo reason impossible.